Examples/JamitFoundationExample/App/Sources/ListView/ListItem/ListItemViewModel.swift

```diff
@@ -0,0 +1,18 @@
+//  Copyright © 2020 Jamit Labs GmbH. All rights reserved.
+
+import Foundation
+import JamitFoundation
+
+struct ListItemViewModel: ViewModelProtocol {
+    let title: String?
+
+    init(title: String? = ListItemViewModel.default.title) {
```

Comment:
You can replace `ListItemViewModel` with `Self`

Reply:
Done.